torchgeo/datasets/advance.py

```diff
@@ -199,7 +199,7 @@ def _load_target(self, path: str) -> Tensor:
                 "scipy is not installed and is required to use this dataset"
             )
 
-        array = wavfile.read(path)[1]
+        array = wavfile.read(path, mmap=True)[1]
```

Comment:
Without this I was seeing an issue with older scipy/torch:
```
tests/datasets/test_advance.py::TestADVANCE::test_getitem
  /home/adam/torchgeo/torchgeo/datasets/advance.py:203: UserWarning: The given NumPy array is not writeable, and PyTorch does not support non-writeable tensors. This means you can write to the underlying (supposedly non-writeable) NumPy array using the tensor. You may want to copy the array to protect its data or make it writeable before converting it to a tensor. This type of warning will be suppressed for the rest of this program. (Triggered internally at  /pytorch/torch/csrc/utils/tensor_numpy.cpp:180.)
    tensor = torch.from_numpy(array)
```
It doesn't seem to be needed for newer scipy/torch, but as long as the tests pass I don't see any harm in always using this.